src/docs/release/breaking-changes/container-color.md

```diff
@@ -89,9 +89,9 @@ Relevant issues:
 Relevant PRs:
 * [Colored box and container optimization #50979][]
 
-[`Container`]: {{site.api}}/flutter/widgets/Container-class.html
-[`ColoredBox`]: {{site.api}}/flutter/widgets/ColoredBox-class.html
-[`BoxDecoration`]: {{site.api}}/flutter/painting/BoxDecoration-class.html
+[`Container`]: https://master-api.flutter.dev/flutter/widgets/Container-class.html
+[`ColoredBox`]: https://master-api.flutter.dev/flutter/widgets/ColoredBox-class.html
+[`BoxDecoration`]: https://master-api.flutter.dev/flutter/widgets/BoxDecoration-class.html
```

Comment:
Do we have a way to indicate to users when we are linking to non-stable versions of the api docs?

Reply:
Nope. It's never been an issue before but, now with breaking changes living on our site, it has become an issue. Feel free to file a bug against the infrastructure to identify this in some way, but please don't hold up this PR.

Reply:
I wasn't talking about infrastructure -- I meant something like:

## References

{{site.alert.note}}
The following references link to api docs for non-stable versions of Flutter.
{{site.alert.